djproxy/views.py

```diff
@@ -135,6 +135,10 @@ def proxy(self):
         headers['X-Forwarded-Host'] = self.request.get_host()
         headers['X-Forwarded-For'] = self.xff
 
+        for prefix, url in self.reverse_urls:
+            if url == self.proxy_url and self.request.path.startswith(prefix):
```

Comment:
Nevermind, `proxy_url` doesn't seem to include the path. That's why I'm checking with `==`. So I think this is working. I need to take another look at my look at my local application. I think the problem may be there and not in djproxy.